### PR TITLE
Story/disruptions 340

### PR DESCRIPTION
**Pull request by DeannaSharma**
- created openapi spec for disruptions api
- created frontend page to display swagger view for disruptions
- setup API backend to call disruptions endpoint

### CODE REVIEW

File: transit_odp/api/tests/test_disruptions_api.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
+from datetime import timedelta
+from unittest.mock import MagicMock, patch
+
+import requests
+from lxml.etree import Element, SubElement, tostring
+from requests import Response
+from rest_framework.status import HTTP_200_OK, HTTP_400_BAD_REQUEST, HTTP_406_NOT_ACCEPTABLE
```
Comment (alainajaffer) [inline transit_odp/api/tests/test_disruptions_api.py:7]: linting issues would reformat config/settings/base.py
would reformat transit_odp/api/tests/test_disruptions_api.py
would reformat transit_odp/api/views/__init__.py
would reformat transit_odp/api/urls.py
would reformat transit_odp/api/views/disruptions.py
would reformat transit_odp/api/views/avl.py

Run python -m black <filepath>

File: transit_odp/api/views/avl.py [hunk 2]
```diff
@@ -152,22 +152,3 @@ def _get_consumer_api_response(url: str, query_params: QueryDict):
     return content, response_status
 
 
-def create_xml_error_response(error_msg, error_code):
```
Comment (alainajaffer) [inline transit_odp/api/views/avl.py:155]: was the supposed to be removed?

File: transit_odp/api/tests/test_disruptions_api.py (hunk 1 above)
Comment (DeannaSharma) [reply transit_odp/api/tests/test_disruptions_api.py]: fixed

File: config/.env.local.template [hunk 3]
```diff
@@ -78,3 +78,7 @@ MS_CLIENT_ID=
 OTC_CLIENT_SECRET=
 OTC_API_KEY=
 OTC_API_URL=
+
+# Disruptions SIRI API
+SIRI_API_URL=__change_me__
```
Comment (alainajaffer) [inline config/.env.local.template:83]:
```suggestion
SIRI_API_URL=
```
To be consistent with the rest of the file

File: config/.env.local.template [hunk 4]
```diff
@@ -78,3 +78,7 @@ MS_CLIENT_ID=
...
+SIRI_API_KEY=__change_me__
```
Comment (alainajaffer) [inline config/.env.local.template:84]:
```suggestion
SIRI_API_KEY=
```

File: config/.env.local.template (hunk 3 above)
Comment (DeannaSharma) [reply config/.env.local.template]: fixed

File: config/.env.local.template (hunk 4 above)
Comment (DeannaSharma) [reply config/.env.local.template]: fixed

**Review by alainajaffer — APPROVED** [review]

**Pull request merged**